examples/src/bin/unbuffered-client.rs

```diff
@@ -0,0 +1,281 @@
+use std::error::Error;
```

Comment:
This example should start with a comment which explains what it does.

Generally I think example code should be focused on demonstrating one feature or API, to the exclusion of others (even if it is convenient for one-off testing in other scenarios, ideally those tests should be automated). I think the focus of this example should be `converse()`, and we should minimise the distractions in the remainder as far as possible.

Below, I think:
- the commented-out code should be removed,
- `CERTFILE` and the related branch in `build_root_store` (at which point `build_root_store` can be inlined into its caller, and the start of `main()` will be identical to `simpleclient.rs`.)

Reply:
inlining `converse` wouldn't be practical as it is called twice when introducing early data in the unbuffered client example. I'm happy to hear any suggestions about the name.

Reply:
> This example should start with a comment which explains what it does.

I think this feedback is still unaddressed.